api/applicant/controllers/applicant.js

```diff
@@ -41,12 +48,32 @@ module.exports = {
       commitment,
       accepted,
       reason,
-      project: applicantProject.id,
+      project: applicantProject ? applicantProject : null,
```

Comment:
Does that mean `applicantProject` can be null?

Reply:
based on my understanding, it's possible. since the field of 'project' is not a required field in the applicant schema.

Reply:
Hmm I think we should make project required. Otherwise we don't which project lead to notify.

Reply:
Should we skip sending notification if the applicant didn't specify a project? We won't be able to know which project lead to notify.

Reply:
Oh I see  you already check it below in `if (applicantProject)`